lib/all-tools.js

```diff
@@ -9,7 +9,7 @@ export const allTools = {
   importer_cli: 'GitHub Enterprise Importer CLI',
   javascript: 'JavaScript',
   jetbrains: 'JetBrains IDEs (Beta)',
-  neovim: 'Neovim',
+  vimneovim: 'Vim/Neovim',
```

Comment:
We're good on the addition of Vim/Neovim as a tool for the picker from an Engineering POV. However, I think we'll still need to get a review on the Copilot-specific content changes @cmwilson21

Reply:
Thank you for getting this opened @ychin!

Reply:
I'm not really sure who's responsibility it is to do that? Should I ping someone to do so or someone on your team will do so?